draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2046,13 +2048,17 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    the sender &MUST; generate those timestamps in the IMF-fixdate format.
 </t>
 <t>
-   An HTTP-date value represents time as an instance of Coordinated Universal
-   Time (UTC). The first two formats indicate UTC by the three-letter
-   abbreviation for Greenwich Mean Time, "GMT", a predecessor of the UTC name;
-   values in the asctime format are assumed to be in UTC.
+   An HTTP-date value represents time as an instance of Coordinated
+   Universal Time (UTC). The first two formats indicate UTC by the
+   three-letter abbreviation for Greenwich Mean Time, "GMT", a predecessor
+   of the UTC name; values in the asctime format are assumed to be in UTC.
+</t>
+<t>
+   A <x:dfn>clock</x:dfn> is an implementation capable of providing a
+   reasonable approximation of the current instant in UTC.
    A sender that generates HTTP-date values from a local clock ought to use
```

Comment:
```suggestion
   A sender that generates HTTP-date values from a local clock <xref target="RFC6919" section="4">ought to</xref> use
```

Reply:
Nope, I don't like all-caps. Note that the example OUGHT TO is a reference to RFC2616, though we use it more like a REALLY SHOULD.